src/languages/vbnet.js

```diff
@@ -102,6 +109,18 @@ export default function(hljs) {
     ]
   });
 
+  const COMMENT = hljs.COMMENT(null, /$/, {
+    variants: [
+      {
+        begin: /'(?!'')/
+      },
+      {
+        // TODO: Use `beforeMatch:` for leading spaces
+        begin: /([\t ]|^)REM(?!\S)/
```

Comment:
Hmmm... How is "not not space" clearer than "space". IE, `\s`?

Reply:
Oh wait... thinking.

Reply:
Yeah, why not a positive lookahead matching \s?

Reply:
Because it's late. 😆 

Removed the double negative.

Thanks for the quick feedback. If you have more comments, I'll take care of them tomorrow.